lib/trip_list_widget.dart

```diff
@@ -0,0 +1,97 @@
+import 'dart:core';
+
+import 'package:backtrip/service/user_service.dart';
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+
+import 'model/trip.dart';
+
+class TripList extends StatefulWidget {
+
+  @override
+  _TripListState createState() => _TripListState();
+}
+
+class _TripListState extends State<TripList> {
+  Future<List<Trip>> futureTrips;
+
+  @override
+  void initState() {
+    super.initState();
+    futureTrips = UserService.getTrips(1); // TODO: Retrieve user id dynamically
+  }
+
+  Widget tripCard(Trip) {
+    return Padding(
+      padding: const EdgeInsets.all(5.0),
+      child: Card(
+        child: Column(
+          children: [
+            Image.asset(
```

Comment:
Afin d'utiliser des icones, images, fichiers de config... il les spécifier dans le fichier pubspec.yaml, par exemple : 
```
flutter:
  assets:
    - directory/
```
J'ai créé les répertoires assets/images afin de stocker une image de voyage par défaut.
https://flutter.dev/docs/development/ui/assets-and-images